platform/exit.go

```diff
@@ -55,9 +58,12 @@ var defaultExitCodes = map[LifecycleExitError]int{
 	// rebase phase errors: 70-79
 	RebaseError: 72, // RebaseError indicates generic rebase error
 
-	// generate phase errors: 80-89
-	FailedGenerateWithErrors: 81,
-	GenerateError:            82, // GenerateError indicates generic generate error
+	// launch phase errors: 80-89
+	LaunchError: 82, // LaunchError indicates generic launch error
+
+	// generate phase errors: 90-99
+	FailedGenerateWithErrors: 91, // FailedGenerateWithErrors indicates extension error during /bin/generate
```

Comment:
Do we document these somewhere? probably worth calling these changes out in the release notes for platform operators.